### ISSUE TITLE
Support detecting `node --test-reporter=@voxpelli/node-test-pretty-reporter`

### ISSUE THREAD

**Issue by voxpelli** [webpro/knip#171]
When using a module as a custom reporter for `node --test` its currently not detected.

The full npm script definition that I'm using:

```json
"test:node": "c8 --reporter=lcov --reporter text node --test --test-reporter=@voxpelli/node-test-pretty-reporter",
```

Currently goes undetected:

```
Unused devDependencies (1)
@voxpelli/node-test-pretty-reporter  package.json
```

Not a biggie, but thought I would mention it anyhow. Thanks for the continued awesome work on knip!

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.17.1. See [Release 2.17.1](https://github.com/webpro/knip/releases/tag/2.17.1) for release notes._

**Comment by webpro** [comment]
Thanks, Pelle!
